pennylane/gradients/general_shift_rules.py

```diff
@@ -48,25 +51,21 @@ def _process_shifts(rule, tol=1e-10):
     # remove columns where the coefficients are 0
     rule = rule[:, ~(rule[0] == 0)]
 
-    # sort columns according to abs(shift)
-    rule = rule[:, np.argsort(np.abs(rule)[-1])]
-
-    # determine unique shifts
-    round_decimals = int(-np.log10(tol))
-    rounded_rule = np.round(rule[-1], round_decimals)
-    unique_shifts = np.unique(rounded_rule)
+    if not no_duplicates:

```

Comment:
This makes me wonder if the kwarg should be called `allow_duplicates` :)

Reply:
Oh! I understand better. `contains_duplicates`?

Reply:
Ah, yes! That sounds great!